server/prpc/server.go

```diff
@@ -111,8 +111,7 @@ func (s *Server) authenticate() router.Middleware {
 	if a == nil {
 		a = GetDefaultAuth()
 		if a == nil {
-			panic("prpc: no custom Authenticator was provided and default authenticator was not registered. " +
```

Comment:
I'd actually prefer we leave the panic in. The rationale here is that if someone explicitly wants no auth, they can install an empty default auth (`[]auth.Authenticator(nil)`), but if they forget to install auth, that's a problem worth panicking over.